vembrane/common.py

```diff
@@ -32,7 +32,10 @@ def get_annotation_keys(header: VariantHeader, ann_key: str) -> List[str]:
             continue
         if rec.get("ID") == ann_key:
             return list(
-                map(str.strip, rec.get("Description").split(separator)[1].split("|"))
+                map(
+                    str.strip,
+                    rec.get("Description").strip('"').split(separator)[1].split("|"),
```

Comment:
Is the content of `Description` somewhat well-formed/restricted? Or could it also be something like
`"Description": '"Foo: Yadda yadda "quote""'` <- having an additional `"` before the end double quote.

Reply:
it's well formed, as far as I know.